recipes/ffmpeg/bld.bat

```diff
@@ -0,0 +1,3 @@
+:: http://ffmpeg.zeranoe.com/blog/?p=300
+
+bash "%RECIPE_DIR%\mingw-w64-build-3.6.0.sh"
```

Comment:
Binaries of ffmpeg. Just until we figure out how we should build it.

Yeah, it was my understanding that conda's MinGW was a bit broken. Are you saying you have a different way of shipping the relevant MinGW bits?